app/main.py

```diff
@@ -30,3 +50,64 @@ def profile(request: Request):
         "username": current_username,
         "events": upcouming_events
     })
+
+
+@app.get("/profile/{user_id}/EditEvent", response_class=HTMLResponse)
+async def insert_info(request: Request):
+    """Get request and return an html File"""
+    return templates.TemplateResponse("editevent.html",{"request": request})
+
+
+@app.post("/profile/{user_id}/EditEvent") # this func is soupose to change with the PR of Ode and Efrat and it will be change
+def create_event(user_id: int, event_title: str = Form(...), location: Optional[str] = Form(None), from_date: Optional[datetime] = Form(...),
+                to_date: Optional[datetime] = Form(...), link_vc: str = Form(None), content: str = Form(None),
+                 db = Depends(get_db)):
+    """ required args - title, from_date, to_date, user_id, the 'from_date' need to be early from the 'to_date'.
+    check validation for the value, insert the new data to DB 
+    if the prosess success return True arg the event item, otherwith return False and the error msg """
+    success = False
+    error_msg = ""
+    new_event = ""
+    event_value = {'title': event_title, "location": location, "start_date": from_date, "end_date": to_date, "VC_link":link_vc, "content": content, 
+                    "owner_id": user_id}
+    if event_title is None:
+        event_title = "No Title"
+    try:
+        if check_validation(from_date, to_date):
+            new_event = add_event(event_value, db)
+            success = True
+        else:
+            error_msg = "Error, Your date is invalid"
+    except Exception as e:
+        error_msg = e
+    finally:
+        return {success, new_event, error_msg}
+
+
+def check_validation(start_time, end_time):
+    """Check if the start_date is smaller then the end_time"""
+    if start_time < end_time:
```

Comment:
`return start_time < end_time`

Reply:
In the docstring you wrote start_date , it should be strart_time.
Also I would suggest writing earlier and not smaller.